src/data/data.hpp

```diff
@@ -83,15 +83,15 @@ struct TemperatureData : public Sensor {
 };
 
 struct Sensors : public Module {
-  static constexpr int kNumImus = 4;
-  static constexpr int kNumKeyence = 2;
+  static constexpr int kNumImus = 4;      // TODO(anyone): read in from config
```

Comment:
Please, remove these completely. Or is this something to be done later? At the moment, it seems config stuff is spread all over the place, some in `data` some in `config`. Could you move it all to `config`?

Reply:
I did not know whether we are going to do everything from config or data. Won't be my decision/responsibility to take it out of data.hpp without approval, that's why I just added a comment. Do you reckon take it out?

Reply:
My suggestion is to put everything that can change during testing, development, or in Spacex parking lot, e.g. number of IMUs, into config. Double check with heads if uncertain.

Reply:
Going to talk with state machine in regards to these, waiting for their response to see how it is best to approach this, but those changes should be independent of this pr since its a bigger change in data.hpp.